src/components/sidenav/SidebarDesktop.vue

```diff
@@ -126,12 +118,12 @@ export default defineComponent({
   setup() {
     const { isOpen } = useSidebar();
     const { isZkEvm, isZkyoto } = useNetworkInfo();
+    const { isEnabled: isGovernanceEnabled, homepageUrl } = useGovernance();
```

Comment:
imo, it's better to use `governanceUrl`

Reply:
Fixed in https://github.com/AstarNetwork/astar-apps/pull/1402/commits/4d7e1095910a053bf8ffe8d3e9a851284ee42368